tests/fixtures/snapshot_date.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+from typing import Any
+
+import pytest
+from syrupy.matchers import path_type
+
+
+def validate_time(timestamp: datetime.datetime | Any):
+    """
+    Validates the format and recency of the timestamp
+    """
+    if not isinstance(timestamp, datetime.datetime):
+        raise ValueError("timestamp in wrong format")
+
+    if datetime.datetime.utcnow() - timestamp < datetime.timedelta(seconds=30):
+        return "approximately_now"
+    return "not_approximately_now"
+
+
+@pytest.fixture
+def snapshot_recent(snapshot):
+    return snapshot.with_defaults(
```

Comment:
* Extend this so that if a `str` is found for the path, we try to parse it as an ISO formatted time using Arrow. If this fails, we can throw a `ValueError`.

  The representation of the time should show up in the snapshot value. This will allow us to avoid a ISO string getting put in MongoDB instead of a `datetime`. For example, `approximately_now_isoformat` or `approximately_now_datetime`.

  If the value is not a `datetime` or `str`, we should throw `TypeError`. I think it's best to limit the scope of this fixture to time representations we actually use.
* Add docstring describing usage that is suitable for guiding devs using this fixture.